### DISCUSSION TITLE
API Gridpoints test Dec 5 @ 1230Z

### DISCUSSION THREAD

**Discussion by dwhitemv25** [weather-gov/api#783] [General]
The SCN regarding the gridpoints outage was updated this morning:

https://www.weather.gov/media/notification/pdf_2023_24/pns24-66_potential_api_outage_aab.pdf

There will be a 1 hour test of a solution to the gridpoints issue Dec 5 @ 1230Z. Issues should be report to nco.ops@noaa.gov.

**Comment by jah1007** [comment]
A new PNS should be coming out today or Monday but ultimately, the test that was executed yesterday was successful. The software update that will fix the end-of-the-year bug will be made operational ~1500Z (10A ET) on Thursday 12/12.

**Reply by jah1007** [reply]
Shortly after 10 ET today this was activated on our production system. We are seeing positive stability and no indication of issues. Any issues should be reported to nco.ops@noaa.gov.

**Reply by dwhitemv25** [reply]
Last 500 error recorded here was 2024-12-12 @ 9:30am EST, before the changeover. Will continue to monitor. I hit the endpoint once an hour.
